modules/op-release-notes-1-2.adoc

```diff
@@ -0,0 +1,104 @@
+* Proper validation for replicas in the `EventListener` specification is now available for cases when a user specifies the `spec.replicas` values as `negative` or `zero`.
+* You can now specify the `TriggerCRD` object inside the `EventListener` spec as a reference using the `TriggerRef` field to create the `TriggerCRD` separately and then bind it inside the `EventListener` spec.
+* Validation and defaults for the `TriggerCRD` object are now available.
+
+[id="deprecated-features-1-2_{context}"]
+== Deprecated features
+
+* `$(params)` are now deprecated in favor of  `$(tt.params)` to avoid confusion between the `resourcetemplate` and `triggertemplate` parameters.
+* The `ServiceAccount` reference of the optional `EventListenerTrigger` based level of authentication has changed from an object reference to a string `ServiceAccountName`. This ensures that the `ServiceAccount` is in the same namespace as the `EventListenerTrigger`.
+* The `Conditions` CRD are now deprecated, use `WhenExpressions` instead.
+* The `PipelineRun.Spec.ServiceAccountNames` is being deprecated in favor of `PipelineRun.Spec.TaskRunSpec[].ServiceAccountName`.
+
+[id="known-issues-1-2_{context}"]
+== Known issues
+
+* This release of {pipelines-title} adds support for a disconnected installation. However, some images used by the cluster tasks need to be mirrored in order to work in disconnected clusters.
+* Pipelines in the `openshift` namespace are not deleted after you uninstall the {pipelines-title} Operator. Use the `oc delete pipelines -n openshift --all` command to delete the pipelines.
+* Uninstalling the {pipelines-title} Operator does not remove the event listeners.
++
+As a workaround, to remove the `EventListener` CRD and `Pod`:
++
+. Edit the `EventListener` object with the `foregroundDeletion` finalizers:
++
+----
+oc patch el/<eventlistener_name> -p '{"metadata":{"finalizers":["foregroundDeletion"]}}' --type=merge
+----
++
+For example:
++
+----
+oc patch el/github-listener-interceptor -p '{"metadata":{"finalizers":["foregroundDeletion"]}}' --type=merge
+----
++
+. Delete the `EventListener` CRD:
++
+----
+oc patch crd/eventlisteners.triggers.tekton.dev -p '{"metadata":{"finalizers":[]}}' --type=merge
+----
+
+[id="fixed-issues-1-2_{context}"]
+== Fixed issues
+
+* A simple syntax validation to check the CEL filter, overlays in the Webhook validator, and the expressions in the interceptor has now been added.
+* Triggers no longer overwrite annotations set on the underlying deployment and service objects.
+* Previously an event listener would stop accepting events. This fix adds an idle timeout of 120 seconds for the `EventListener` sink to resolve this issue.
+* Previously, cancelling a `PipelineRun` with a `Failed(Cancelled)` state gave a success message, this has been fixed to display an error instead.
```

Comment:
"a `PipelineRun`" what? Probably should specify `PipelineRun` process/object/task, or lowercase as "pipeline run".

Reply:
s/success message, this has been/success message. This has been/